bitcoin/src/blockdata/script.rs

```diff
@@ -615,8 +615,8 @@ macro_rules! delegate_index {
 }
 
 delegate_index!(Range<usize>, RangeFrom<usize>, RangeTo<usize>, RangeFull, RangeInclusive<usize>, RangeToInclusive<usize>);
-#[cfg(feature = "rust_v_1_53")]
-#[cfg_attr(docsrs, doc(cfg(feature = "rust_v_1_53")))]
+#[cfg(rust_v_1_53)]
+#[cfg_attr(docsrs, doc(cfg(rust_v_1_53)))]
```

Comment:
Oh, crap, this caused the information about the feature only available in Rust 1.53+ disappear from the doc because rustdoc doesn't understand custom flags.

I wonder if I had the foresight when I implemented it or it was by luck. I can't remember.

Reply:
Hmm, yeah, crap. We may need to add this manually.